src/lib/__tests__/messenger.test.js

```diff
@@ -107,8 +107,9 @@ describe('Messenger', () => {
     });
 
     describe('when asking for tourettes', () => {
-      const tourettesCommand = '/tourettes';
-      const msg = Object.assign({}, message, { text: tourettesCommand });
+      const tourettesInputText = '/tourettes Hej';
```

Comment:
The naming is kind of in reverse, the `tourettesCommand` should include the actual command in my opinion, i.e.

```js
const tourettesInputText = 'tourettes input text'; // Note: Prefer to declare the test variables with the name of the variable itself.
const tourettesCommand = `/tourettes ${tourettesInputText}`;
```